attributes/events/observer.md

```diff
@@ -0,0 +1,31 @@
+# Observer
+
+The `Observer` attribute allows you to register a observer class for model events.
+
+This is used to register a Observer Class with a model explained in more detail here: (Laravel Docs: Eloquent Observers)[https://laravel.com/docs/10.x/eloquent#observers]
+
+## Usage
+
+```php
+use Illuminate\Database\Eloquent\Model;
+use WendellAdriel\Lift\Attributes\Events\Observer;
+use WendellAdriel\Lift\Lift;
+
+
+#[Observer(ProductObserver::class)]
+final class Product extends Model
+{
+    use Lift;
+
+}
+```
+
+## API
+
+```php
+#[Observer(string $observer)]
+```
+
+Properties:
```

Comment:
Put this and below in text format not list, because you don't need to add the name or the type of the parameters since it's already described above in the API example.